PR_GUIDELINES.md

```diff
@@ -47,6 +47,7 @@ npm run format:check
 1. Please read our [CONTRIBUTING.md](CONTRIBUTING.md) document for details on our testing policy.
 
 ## Pull Request Processing
+
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project